docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,197 @@
+
+## Internal vs external users
+
+As a first step, determine how you want to manage users who will access Grafana.
+
+Do you already use an identity provider to manage users? If so, Grafana might be able to integrate with your identity provider through one of our IdP integrations.
+Refer to [Configure authentication documentation]({{< relref "./configure-authentication" >}}) for the list of currently supported providers.
+
+If you are not interested in setting up an external identity provider, but still want to limit access to your Grafana instance, you should consider using Grafana's basic authentication.
+
+Finally, if you want your Grafana instance to be accessible to everyone, you can enable anonymous access to Grafana.
+For information, refer to the [anonymous authentication documentation]({{< relref "./configure-security/configure-authentication#anonymous-authentication" >}}).
+
+## Ways to organize users
+
+Organize users in subgroups that are sensible to the organization. These are some examples:
+
+- **Security**: Different groups of users or customers should only have access to their intended resources.
+- **Simplicity**: Reduce the scope of dashboards and resources available.
+- **Cost attribution**: Track and bill costs to individual customers, departments, or divisions.
+- **Customization**: Each group of users could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana teams
+
+You can organize users into [teams] and assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team.
+
+A user can belong to multiple teams, and can be a member or an administrator for a given team. Members of a team inherit permissions from the team, but they cannot edit the team itself. Team administrators can add members to a team and update its settings, such as the team name, team members, roles assigned to the team, and UI preferences.
+
+Teams are a perfect solution for working with a small subset of users. Teams can share resources with other teams.
+
+### Users in Grafana organizations
+
+[Grafana organizations] allow complete isolation of resources, such as dashboards and data sources. Users can be members of one or several organizations, and they can only access resources from an organization that they belong to.
+
+Having multiple organizations under a single instance of Grafana allows you to manage your users in one place, while having a complete separation of resources.
+
+Organizations provide a higher measure of isolation within Grafana than teams do, and can be useful in certain scenarios. However, because organizations lack the scalability and flexibility of teams and [folders], we do not recommend using them as the default way to group users and resources.
```

Comment:
```suggestion
Organizations provide a higher measure of isolation within Grafana than teams do and can be helpful in certain scenarios. However, because organizations lack the scalability and flexibility of teams and [folders], we do not recommend using them as the default way to group users and resources.
```